flow/project.py

```diff
@@ -95,6 +97,13 @@
 {filters}"""
 
 
+class Ignore_condition(IntEnum):
```

Comment:
```suggestion
class IgnoreConditions(IntEnum):
```
Classes should **always** be camel case.